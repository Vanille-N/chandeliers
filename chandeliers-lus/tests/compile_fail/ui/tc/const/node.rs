//! Trying to use a function call in a `const` context.
//! Due to historical reasons this is a duplicate of `const/call`.
chandeliers_lus::decl! {
    const X : float = test(5);
}

fn main() {}
