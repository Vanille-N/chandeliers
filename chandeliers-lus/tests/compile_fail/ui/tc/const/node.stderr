error: Function calls are not valid in const contexts
 --> tests/compile_fail/ui/tc/const/node.rs:4:5
  |
4 |     const X : float = test(5);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: You must put this definition inside a node
